runtime/compiler/optimizer/InlinerTempForJ9.cpp

```diff
@@ -884,11 +884,14 @@ TR_J9InlinerPolicy::genCodeForUnsafeGetPut(TR::Node* unsafeAddress,
    TR::TreeTop* nullComparisonTree = nullComparisonBlock->getLastRealTreeTop();
    TR::TreeTop *nullComparisonEntryTree = nullComparisonBlock->getEntry();
    TR::TreeTop *nullComparisonExitTree = nullComparisonBlock->getExit();
```

Comment:
high level:  blocks layout is considered already (normal object is the most likely case)?  i.e.  directAccess should be the first block, then arrayAccess, then indirectAccess.